prow/cmd/deck/static/script.js

```diff
@@ -489,6 +489,17 @@ function redraw(fz) {
         }
         var r = document.createElement("tr");
         r.appendChild(stateCell(build.state));
+        if (spyglass && build.url.startsWith("https://k8s-gubernator.appspot.com/build/")) {
+            const icon = createIcon("visibility", "View in Spyglass");
+            icon.href = build.url.replace("https://k8s-gubernator.appspot.com/build/",
+                "https://prow.k8s.io/view/gcs/");
+            const cell = document.createElement("TD");
+            cell.classList.add("icon-cell");
+            cell.appendChild(icon);
+            r.appendChild(cell);
+        } else {
+            r.appendChild(createTextCell(""));
```

Comment:
if not spyglass, then don't append the cell? ( just put the entire block inside `if (spyglass)`)